apps/dashboard/app/javascript/launcher_edit.js

```diff
@@ -19,6 +19,10 @@ const newFieldData = {
     label: "Job Name",
     help: "The name the job will have."
   },
+  auto_output_directory: {
+    label: "Log Output Directory",
+    help: "The destination of the job's log output."
```

Comment:
This is one comment, but applicable to a lot in this PR. We're not specifying the log directory, we're specifying the output log itself. I.e., it's a file, not a directory.